src/server/location.rs

```diff
@@ -32,11 +40,13 @@ impl From<&RequestHead> for LocationResult {
         if let Some(loc_string) = headers.get(GOOG_LOC_HEADER) {
             dbg!("Found google header", loc_string);
             let mut parts: Vec<&str> = loc_string.to_str().unwrap_or("").split(',').collect();
-            return Self {
-                city: parts.pop().map(ToOwned::to_owned),
-                country: parts.pop().map(ToOwned::to_owned),
+            let reply = Self {
+                country: parts.pop().map(|v| v.trim().to_owned()),
```

Comment:
Also note the changes here I made in #78 --

- don't want city (yet), just country and subdivision
- it's gonna be "--custom-request-header 'X-Client-Geo-Location: {client_region},{client_region_subdivision}"
- `client_region` is mostly country. `client_region_subdivision` is a "Unicode CLDR subdivision ID" which includes a 2 letter country prefix we strip

Reply:
I'll correct the header we get location from.